docs/README.rst

```diff
@@ -4,6 +4,8 @@ Documentation for boto3 can be found `here <https://boto3.amazonaws.com/v1/docum
 
 Generating Documentation
 ~~~~~~~~~~~~~~~~~~~~~~~~
+Note: `Botocore's <https://github.com/boto/botocore>`_ requirement-docs.txt must be installed prior to attempting the following steps.
```

Comment:
minor nit; It may be more useful to link directly to the file like:

```
Botocore's `requirement-docs.txt <https://github.com/boto/botocore/blob/develop/requirements-docs.txt>`_
```